bokehjs/src/lib/models/annotations/legend.ts

```diff
@@ -413,4 +426,14 @@ export class Legend extends Annotation {
     }
     return legend_names
   }
+
+  get_max_glyph_size(): number {  // should this return a Size? but Size is 2d, which seems unnecessary
```

Comment:
This could probably a property getter so that things can be spelled more simply: `this.model.max_glyph_size`

Reply:
Done!